### DISCUSSION TITLE
How to generate DB with entity first method?

### DISCUSSION THREAD

**Discussion by frederikhors** [SeaQL/sea-orm#1031] [Q&A] [answered]
Hi guys! I cannot find anywhere (docs/tutorial) how to create DB from entitites files instead of schema (migration) first method.

I'm used to declare Golang struct like:

```go
type Player struct {
  Name string
  TeamID uint64
}
```

Is it possible with SeaORM?

I only found out this: https://www.sea-ql.org/SeaORM/docs/migration/writing-migration/#schema-first-or-entity-first.

**But I cannot understand how to do it.**

**Comment by frederikhors** [comment, marked as answer]
@billy1624 answered me on Discord. Thanks. I opened https://github.com/SeaQL/seaql.github.io/pull/52.

**Reply by billy1624** [reply]
No problem! :)
